package.json

```
@@ -16,6 +16,7 @@
     "@voiceflow/verror": "^1.1.0",
     "chai": "^4.3.4",
     "dotenv": "^10.0.0",
+    "gaxios": "^4.3.2",
```

we should probably keep it as a definite version
```suggestion
    "gaxios": "4.3.2",
```

---

don't do this, we want to keep the dependency range wide so that we avoid duplicate installs of `gaxios`